testing/src/main/java/net/consensys/linea/testing/ReplayExecutionEnvironment.java

```diff
@@ -195,7 +195,7 @@ private static void executeFrom(
     MutableWorldState world = initWorld(conflation);
     // Construct the transaction processor
     final MainnetTransactionProcessor transactionProcessor =
-        ExecutionEnvironment.getProtocolSpec(chain.id).getTransactionProcessor();
+        ExecutionEnvironment.getProtocolSpec(chain.id, "london").getTransactionProcessor();
```

Comment:
Why do we use
```java
fork = "london"
```
in `ReplayExecutionEnvironment.java` and
```java
fork = "shanghai"
```
in `ToyExecutionEnvironmentV2.java` ?